frontend/pkg/frontend/middleware_logging.go

```diff
@@ -71,13 +73,45 @@ func MiddlewareLogging(w http.ResponseWriter, r *http.Request, next http.Handler
 	logger = logger.With(
 		"request_method", r.Method,
 		"request_path", r.URL.Path,
+	)
+
+	// make a best attempt at parsing the resourceID. This will often fail because we have non-resource requests.
+	// we do this so that we can add subscription, resourceGroup, and hcpCluster to the logger context for future searching
+	// if possible.
+	// It's important to do before the second panic handler so that panics can be correlated easily.
+	// TODO are the value we find case sensitive or case insensitive.  They used to be case sensitive, so I have left that
```

Comment:
if they aren't case sensitive, then our search is much easier if we tolower everything